docs/customize/administration_interface.md

```diff
@@ -0,0 +1,398 @@
+# Administration panel views
+
+## Summary
+
+The following document is a developer guide to the invenio administration panel, detailing its programmatic interface and usage.
```

Comment:
This doc is under `customization`, not topic guide. The first sentence suggests that I will learn about Invenio-administration, in general, but instead it should be about customizing existing implemeantion.